src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -47,6 +43,8 @@ def handle_exception(ex):  # pylint: disable=too-many-return-statements
     from msrestazure.azure_exceptions import CloudError
     from msrest.exceptions import HttpOperationError, ValidationError, ClientRequestError
     from azure.cli.core.azlogging import CommandLoggerContext
+    from azure.common import AzureException
```

Comment:
There is only ONE SDK that is still using `azure.common.AzureException` and it's [storage](https://github.com/Azure/azure-multiapi-storage-python/search?q=AzureException&unscoped_q=AzureException). If that helps, you should be able to move that to storage data-plane specific commands (until you can move to full track2 storage)

Reply:
@Juliehzl @jsntcy

Reply:
yes, as what Laurent said, we are still using AzureExeption with existing track 1 storage commands and we cannot fully move to track 2 now.

Reply:
what I mean is that this import is not necessary in cli-core, if only on particular set of SDKs is using it (just trying to save some more :))